publication-commons/src/main/java/no/unit/nva/publication/permissions/file/deny/HiddenFileDenyStrategy.java

```diff
@@ -3,19 +3,23 @@
 import no.unit.nva.model.FileOperation;
 import no.unit.nva.model.associatedartifacts.file.HiddenFile;
 import no.unit.nva.publication.model.business.FileEntry;
+import no.unit.nva.publication.model.business.Resource;
 import no.unit.nva.publication.model.business.UserInstance;
 import no.unit.nva.publication.permissions.file.FileDenyStrategy;
 import no.unit.nva.publication.permissions.file.FileStrategyBase;
 
 public final class HiddenFileDenyStrategy extends FileStrategyBase implements FileDenyStrategy {
 
-    public HiddenFileDenyStrategy(FileEntry file,
-                                  UserInstance userInstance) {
-        super(file, userInstance);
+    public HiddenFileDenyStrategy(FileEntry file, UserInstance userInstance, Resource resource) {
+        super(file, userInstance, resource);
     }
 
     @Override
     public boolean deniesAction(FileOperation permission) {
-        return file.getFile() instanceof HiddenFile;//TODO add exception of curator on same institution
+        if (currentUserIsFileCuratorForGivenFile()) {
+            return false;//allow curator to access hidden files
+        }
+
+        return file.getFile() instanceof HiddenFile;
```

Comment:
This can be simplified

Reply:
suggestions? find it tricky to read with not-statements inside one-liner with AND. False return in this method is means "success" and that is already too much for my head 😅

```java
 return file.getFile() instanceof HiddenFile && !currentUserIsFileCuratorForGivenFile();    
```